src/pages/dataElements/form/customFields.tsx

```diff
@@ -183,7 +196,9 @@ export function AggregationTypeField() {
     return (
         <FieldRFF
             component={SingleSelectFieldFF}
+            dataTest="dataelementsformfields-aggregationtype"
             required
+            validate={(value: string) => (!value ? 'Required' : undefined)}
             inputWidth="400px"
             name="aggregationType"
             label={i18n.t('Aggregation type (required)')}
```

Comment:
I know this wasnt changed here, but can we make the labels an interpolated value? `i18n.t('{{modelType}} (required)', { model: i18n.t('Aggregation type' })`